src/tools/interop/idt/discovery/dnssd.py

```diff
@@ -31,104 +32,114 @@
 
 
 @dataclass()
-class MdnsTypeInfo:
+class ServiceTypeInfo:
     type: str
     description: str
 
 
-commissioner = MdnsTypeInfo(
+_COMMISSIONER_SERVICE_INFO = ServiceTypeInfo(
     "COMMISSIONER",
-    "This is a service for a Matter commissioner aka. controller"
+    "A service for a Matter commissioner aka. controller"
 )
-commissionable = MdnsTypeInfo(
+_COMMISSIONABLE_SERVICE_INFO = ServiceTypeInfo(
     "COMMISSIONABLE / EXTENDED DISCOVERY",
-    "This is a service to be used in the commissioning process and provides more info about the device."
+    "A service to be used in the commissioning process and provides more info about the device."
 )
-operational = MdnsTypeInfo(
+_OPERATIONAL_SERVICE_INFO = ServiceTypeInfo(
     "OPERATIONAL",
-    "This is a service for a commissioned Matter device. It exposes limited info about the device."
+    "A service for a commissioned Matter device. It exposes limited info about the device."
 )
-border_router = MdnsTypeInfo(
+_TBR_SERVICE_INFO = ServiceTypeInfo(
     "THREAD BORDER ROUTER",
-    "This is a service for a thread border router; may be used for thread+Matter devices."
+    "A service for a thread border router; may be used for thread+Matter devices."
 )
 
-_MDNS_TYPES = {
-    "_matterd._udp.local.": commissioner,
-    "_matterc._udp.local.": commissionable,
-    "_matter._tcp.local.": operational,
-    "_meshcop._udp.local.": border_router,
+_TREL_SERVICE_INFO = ServiceTypeInfo(
+    "THREAD RADIO ENCAPSULATION LINK",
+    "A service for Thread Radio Encapsulation Link which is a method for thread BRs to exchange data on IP links."
+)
+
+_SERVICE_TYPE_INFO = {
+    "_matterd._udp.local.": _COMMISSIONER_SERVICE_INFO,
+    "_matterc._udp.local.": _COMMISSIONABLE_SERVICE_INFO,
+    "_matter._tcp.local.": _OPERATIONAL_SERVICE_INFO,
+    "_meshcop._udp.local.": _TBR_SERVICE_INFO,
+    "_trel._udp.local.": _TREL_SERVICE_INFO,
 }
 
 
 @dataclass()
-class RecordParser:
+class TxtRecordParser:
     readable_name: str
     explanation: str
     parse: Callable[[str], str]
 
 
-# TODO: Meshcop parser
+def unwrap_str(to_unwrap: str) -> str:
+    to_unwrap = to_unwrap.replace("\n", " ")
+    to_unwrap = to_unwrap.replace("\t", " ")
+    return re.sub(' +', ' ', to_unwrap)
+
+
+# TODO: Thread parser
```

Comment:
should there be a github issue linked here?

Reply:
Thanks Andrei I have created [this issue](https://github.com/project-chip/connectedhomeip/issues/30884) to track from the last large review and this one. If possible, I'm happy to have it assigned to me!